docs/dev/development.rst

```diff
@@ -21,41 +21,68 @@ Gafaelfawr is developed by the LSST SQuaRE team.
 Setting up a local development environment
 ==========================================
 
-To develop Gafaelfawr, create a virtual environment with your method of choice (like virtualenvwrapper) and then clone or fork, and install:
+Prerequisites
+-------------
+
+Gafaelfawr uses uv_ for all dependency management.
+A reasonably recent version of :command:`uv` must already be installed.
+See `the uv installation instructions <https://docs.astral.sh/uv/getting-started/installation/>`__ if needed.
+
+Gafaelfawr development requires Docker be installed locally.
+The user doing development must be able to start and manage Docker containers.
+
+Set up development environment
+------------------------------
+
+To develop Gafaelfawr, create a virtual environment with :command:`uv venv` and then run :command:`make init`.
 
 .. prompt:: bash
 
    git clone https://github.com/lsst-sqre/gafaelfawr.git
    cd gafaelfawr
+   uv venv
    make init
 
 This init step does three things:
 
-1. Installs Gafaelfawr in an editable mode with its "dev" extra that includes test and documentation dependencies.
-2. Installs pre-commit, tox, and tox-docker.
+1. Installs Gafaelfawr in a virtualenv in the :file:`.venv` directory, including the dependency groups for local development.
+2. Installs pre-commit_, tox_, and the necessary tox plugins.
 3. Installs the pre-commit hooks.
 
-On macOS hosts, you may also need to run the following in the terminal window where you run ``make init`` and where you intend to run ``tox`` commands:
+On macOS hosts, you may need to run the following before running :command:`make init` and in the same terminal window.
 
 .. prompt:: bash
 
    export LDFLAGS="-L/usr/local/opt/openssl/lib"
 
 Otherwise, OpenSSL isn't on the default linker path and some Python extensions may not build.
+This must be set in any terminal window where you may run :command:`tox`.
+
+Finally, you can optionally enter the Gafaelfawr development virtualenv with:
+
+.. prompt:: bash
+
+   source .venv/bin/activate
+
+This is optional; you do not have to activate the virtualenv to do development.
```

Comment:
I think VS Code struggles to do its intellisense magic if an environment isn't available with the app in it, but I get the sentiment here.

Reply:
I added a note saying that editors with Python integration such as VSCode may work better if you activate the virtualenv before starting them.